### PR TITLE
Homework 1 - Mariel Schwartz

### PR DESCRIPTION
**Pull request by marielschwartz**
Homework 1 - Mariel Schwartz - Simply followed the instructions from the page [Using Git within RStudio](https://cfss.uchicago.edu/git05.html) and the [R Markdown Cheatsheet](https://www.rstudio.com/wp-content/uploads/2016/03/rmarkdown-cheatsheet-2.0.pdf).

### CODE REVIEW

**Comment by hongjinjo** [comment]
Evaluation
----------------------------------------------------
| Topic                       | Excellent | Satisfactory | Needs Work |
|-----------------------------|-----------|--------------|------------|
| **Coding style**            |     v      |              |            |
| **Coding strategy**         |      v     |              |            |
| **Presentation**            |     v      |              |            |
| **Achievement, creativity** |     v      |              |            |
| **Ease of access**          |       v    |              |            |

Remarks: 

* The overall code explored a variety of features in markdown. 
* I learned from her quote that we can use quotation marks in markdown to emphasize and quote.

**Comment by hannamn** [comment]
Evaluation
----------------------------------------------------
| Topic                       | Excellent | Satisfactory | Needs Work |
|-----------------------------|-----------|--------------|------------|
| **Coding style**            |     x      |              |            |
| **Coding strategy**         |    x       |              |            |
| **Presentation**            |      x     |              |            |
| **Achievement, creativity** |     x      |              |            |
| **Ease of access**          |     x      |              |            |

Remarks:
* Your code is so nice! It uses all the required interventions, though the image is not appearing to me, which is a big shame, I wanted to see Isabel!
* I think you're image is not showing because it's a jpg extension. I could only make png work. Not sure if it's just me or if the program only reads/accepts pngs.
* I had no ideas you could open quotes with > which was pretty cool!
* Were you trying to do something with the (@)?

**Comment by audreyyxshi** [comment]
Evaluation
----------------------------------------------------
| Topic                       | Excellent | Satisfactory | Needs Work |
|-----------------------------|-----------|--------------|------------|
| **Coding style**            | x          |              |            |
| **Coding strategy**         |x           |              |            |
| **Presentation**            |           |     x         |            |
| **Achievement, creativity** |x           |              |            |
| **Ease of access**          | x          |              |            |

Remarks:

* I think the overall coding format is good. 
* I really want to see the pic of your daughter but it doesn't show up. I guess it is because either the route of the picture is incorrect or the you didn't save the pic in github folder. 
* I learn from you how to quote (>)!

**Comment by risaman** [comment]
Evaluation
----------------------------------------------------
| Topic                       | Excellent | Satisfactory | Needs Work |
|-----------------------------|-----------|--------------|------------|
| **Coding style**            |      x     |              |            |
| **Coding strategy**         |     x      |              |            |
| **Presentation**            |      x     |              |            |
| **Achievement, creativity** |     x      |              |            |
| **Ease of access**          |     x      |              |            |

Remarks:

* Coding is good and implements a variety of features in markdown 
* You taught me how to quote and include a picture (your daughter is adorable!) 
* I wasn't sure what you were trying to do with the (@)

**Comment by alexanme** [comment]
Evaluation
----------------------------------------------------
| Topic                       | Excellent | Satisfactory | Needs Work |
|-----------------------------|-----------|--------------|------------|
| **Coding style**            |      x     |              |            |
| **Coding strategy**         |     x      |              |            |
| **Presentation**            |      x     |              |            |
| **Achievement, creativity** |       x    |              |            |
| **Ease of access**          |    x       |              |            |

Remarks:

* Appropriately uses many of md's formatting options, though as a list-lover i was somewhat disappointed not to see any lists :(
* Cool block quote! Very clear description of coding process! Cute child!
* You reminded me that i forgot to pull from origin master before pushing :grimacing: 
* The detail on your process was great, but I think it's good practice to separate the *product or result* of a coding project from the *meta commentary*; since the main document was supposed to be a bio, the info on the coding process might have been better placed in the commit messages or pull request comment.
* I was also confused by the @ :/

File: README.md [hunk 1]
```diff
@@ -1,2 +1,33 @@
 # hw01
-Repository for homework 01
+Repository for homework 01  
+Author: Mariel Schwartz  
+
+## Introduction
+
+Hello. My name is Mariel. I am a fourth year Ph.D. student in the Economics Department. My research is on labor economics. I have a 3-month old daughter, Isabel. I love to show people pictures of her.
+
+![Me and Isabel](picture.jpg)
```
Comment (mayank-kgp) [inline README.md:9]: Wow. Too adorable.

File: README.md [hunk 2]
```diff
@@ -1,2 +1,33 @@
...
+
+## How I Made Changes to This File
+
+To make changes to this file, I did the following things:
+
+(@) I forked the repo from the class GitHub page to my GitHub page. What is the difference between fork and pull? According to [GitHub Help](https://help.github.com/articles/fork-a-repo/),
```
Comment (mayank-kgp) [inline README.md:15]: I didn't understand the use of @. Maybe you would have wanted to use * for making a list.

File: README.md [hunk 3]
```diff
@@ -1,2 +1,33 @@
...
+
+![Me and Isabel](picture.jpg)
+
+## How I Made Changes to This File
+
+To make changes to this file, I did the following things:
+
+(@) I forked the repo from the class GitHub page to my GitHub page. What is the difference between fork and pull? According to [GitHub Help](https://help.github.com/articles/fork-a-repo/), 
+
+> "A fork is a copy of a repository. Forking a repository allows you to freely experiment with changes without affecting the original project. Most commonly, forks are used to either propose changes to someone else's project or to use someone else's project as a starting point for your own idea."
```
Comment (mayank-kgp) [inline README.md:17]: It is very appreciable to use internet resources for learning.

**Comment by mayank-kgp** [comment]
Evaluation
----------------------------------------------------
| Topic                       | Excellent | Satisfactory | Needs Work |
|-----------------------------|-----------|--------------|------------|
| **Coding style**            |     x      |              |            |
| **Coding strategy**         |           |       x       |            |
| **Presentation**            |       x    |              |            |
| **Achievement, creativity** |     x      |              |            |
| **Ease of access**          |      x     |              |            |

Remarks:

* Very well designed page. Just a side note - You were supposed to write about _how you changed the readme.md_  in the [description box](https://github.com/uc-cfss/hw01/pull/25#issue-220994740). Good luck for your research.

**Pull request closed without merging**